src/piggui.rs

```diff
@@ -78,10 +80,18 @@ impl Sandbox for Gpio {
             }
         };
 
+        let num_pins = GPIO_DESCRIPTION.len();
+
+        // TODO Fixing default PinFunction for each pin
+        // For now displaying Ground for each pin
+
+        let pin_function_selected = vec![PinFunction::Ground; num_pins];
```

Comment:
I'd initialize it as vec or array of None, at start-up, just before calling GPIOConfig::load() if there is a command line option for a filename, then iterate through the pins and over-write any config of a pin that is read in.

I'd put that into a function, so that later we can re-use it if we add a UI option to "Load config from file"...

Reply:
Got it, will do